lib/capybara/webkit/matchers.rb

```diff
@@ -7,11 +7,11 @@ module RspecMatchers
           actual.error_messages.any?
         end
 
-        failure_message_for_should do |actual|
+        failure_message do |actual|
```

Comment:
Would doing `alias_method :failure_message :failure_message_for_should` be enough? Or should we check for its existence:

```ruby
if respond_to?(:failure_message)
  failure_message do
else
  failure_message_for_should do
end
```